CharacterCreation/02_Classes.txt

```diff
@@ -427,34 +428,97 @@ Skill Proficiencies:
+class feat. The first time each turn that you track an enemy that you already
+tracked this encounter, you may do so as a Free Action instead of an Action.
+	For example, let's say a Hunter was fighting an enemy robot and an enemy 
+soldier. The first turn, the Hunter spends one Action to track the robot. The
+second turn, the Hunter spends an Action to track the soldier. The third turn 
+the Hunter tracks the robot again, but this time it is a Free Action, so the 
+Hunter still has 4 Actions left on his turn. Continuing his third turn, the 
+Hunter also tracks soldier. Even though the Hunter already tracked the soldier 
+before, the hunter has used up his one Free Tracking Action this turn, and 
+tracking the soldier again costs one Action.
+	In addition, teammates now get a bonus to critically hit on tracked enemies 
+that they hit in accordance to their range and weapon type. This is accomplished 
+by you calling out weak spots that you see on the target and describing shooting 
+conditions such as the bullet drop due to humidity, wind speed and direction, 
+and the coriollis effect making the bullet swerve. The bonus to critically hit 
+is listed below. Note that if the chance to critically hit exceeds the chance to 
+hit, it does not increase the chance to hit; it only garuntees that a hit will 
+be a critical hit.
+
++-------+------------+-----------------------------+
+| Range | Crit bonus | Crit bonus with Long Rifles |
++-------+------------+-----------------------------+
+|   R1  |     5%     |             10%             |
++-------+------------+-----------------------------+
+|   R2  |    10%     |             25%             |
++-------+------------+-----------------------------+
+|   R3  |    20%     |            100%             |
++-------+------------+-----------------------------+
+
+Pack Hunter (I):
+	You get a +2 to accuracy rolls shooting at an enemy that is being suppressed 
+by one of your party members. You also get +2 to melee attack rolls against an 
+enemy who is adjacent to one of your party members. Teammates who make a melee
+attack against an enemy that is adjacent to you also get a +2 to their melee 
+attack roll. A party member is a character that is controlled by a player other
+than a DM who has played more than one encounter with your character. 
+
+Pack Hunter (II):
+	Requires that your character be level 10 and that you have Pack Hunter (I).
+Your bonus for shooting at an enemy suppressed by your party member now includes
+a +5% chance to critically hit. The bonus for melee for both you and your 
+teammates increases to +3. 
+
+Bushwhacker:
+	You've had enough toothy critters jump out of the brush and try to bite your
+head off to be pretty good with shooting things at close range. You get 
+advantage on shooting targets that are weilding a melee weapon with your primary 
+weapon within your weapon's R1 range bracket. You also get advantage on 
```

Comment:
This sentence has a lot of (3) subordinate clauses; it can be unwieldy to read and parse a sentence with a lot of consecutive sub. clauses. 
Additional question: does that target have to be attacking with a melee weapon, or can it include CQB weapons and/or melee creature attacks (like bite- or claw-attacks). I can imagine this getting house-ruled pretty quickly to explicitly include creatures and unarmed assailants.